dimod/binary/binary_quadratic_model.py

```diff
@@ -690,16 +714,16 @@ def add_quadratic_from(self, quadratic: Union[Mapping, Iterable]):
         Args:
             quadratic:
                 Collection of interactions and their associated quadratic
-                bias. If a dict, should be of the form `{(u, v): bias, ...}`
-                where `u` and `v` are variables in the model and `bias` is
+                bias. If a dict, should be of the form ``{(u, v): bias, ...}``
```

Comment:
We should either use ``` `` ``` and ``` ` ``` or `:math:` and `:code:`.

Reply:
We made the decision a while ago to go for ``\:code:`` but we still have legacy "`" (and when people copy & paste that can sometimes spread to newer content. I convert whenever I'm working on a file.

Reply:
I may take this back. Since the [shinx docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup) say pretty clearly that we should use ` `` ` for code samples.

Reply:
I do hate it though. Looks so much worse. :shrug:

Reply:
We'll make it part of our CSS update :-)

Reply:
FWIW, backtick syntax is just plain ReST. Nothing to do with sphinx per se. `:code:` and `:math:` as sphinx extensions to the ReST syntax. I prefer the shorter form, but also think it's fine to mix and use whatever seems more appropriate in context.

Reply:
Yes, backtick falls under inline markup. It makes sense to me to usually use it because it's shorter but occasionally it's nice to emphasis by using :code: